packages/amazonq/test/unit/codewhisperer/util/codeParsingUtil.test.ts

```diff
@@ -57,6 +60,90 @@ describe('RegexValidationForJava', () => {
 })
 
 describe('isTestFile', () => {
+    beforeEach(async function () {
+        tempFolder = (await createTestWorkspaceFolder()).uri.fsPath
+    })
+
+    it('validate by file path', async function () {
+        const langs = new Map<string, string>([
+            ['java', '.java'],
+            ['python', '.py'],
+            ['typescript', '.py'],
```

Comment:
Should this be `.ts` and if so is there a flaw with the test if it passes? Does this mapping already exist outside of the tests that can be reused?

Reply:
oh nice callout, checking

Reply:
Nick, it works because this test case is to test "by file path", so as long as the file path includes any string of `tst`, `test`, `tests` then it will return true. 

But `.py` here is definitely a typo and i should fix it.

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/5590/files#diff-4c4d168f15256a5a919ec2fd9fc25c997275401d9bb66bc561c56b2005d900c9R76-R91